gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -125,13 +125,29 @@ def validate(self, parsed_args):
           parsed_args.input_pattern))
 
 
+class AvroWriteOptions(VariantTransformsOptions):
+  """Options for writing Variant records to Avro files."""
+
+  def add_arguments(self, parser):
+    # type: (argparse.ArgumentParser) -> None
+    parser.add_argument('--output_avro_path',
+                        default='',
+                        help='The output path to write Avro files under.')
+
+  def validate(self, parsed_args):
+    # type: (argparse.Namespace, bigquery.BigqueryV2) -> None
+    if not parsed_args.output_table and not parsed_args.output_avro_path:
+      raise ValueError('At least one of --output_table or --output_avro_path '
+                       'options should be provided')
```

Comment:
ah, good to know...i figured it's easier for these kinds of nits to just make the change as a reviewer rather than going back-and-forth....anyhow, this is a beta feature from github so hopefully they'll address these problems.